test/integration/stats_test.lua

```diff
@@ -152,54 +153,62 @@ local eval = {
 }
 
 local simple_operation_cases = {
-    insert = {
+    {
+        name = 'insert',
         func = 'crud.insert',
         args = {
             space_name,
             { 12, box.NULL, 'Ivan', 'Ivanov', 20, 'Moscow' },
         },
         op = 'insert',
     },
-    insert_object = {
+    {
+        name = 'insert_object',
         func = 'crud.insert_object',
         args = {
             space_name,
             { id = 13, name = 'Ivan', last_name = 'Ivanov', age = 20, city = 'Moscow' },
         },
         op = 'insert',
     },
-    get = {
+    {
+        name = 'get',
         func = 'crud.get',
         args = { space_name, { 12 } },
         op = 'get',
     },
-    select = {
+    {
+        name = 'select',
         func = 'crud.select',
         args = { space_name, {{ '==', 'id_index', 3 }} },
         op = 'select',
     },
-    pairs = {
+    {
+        name = 'pairs',
         eval = eval.pairs,
         args = { space_name, {{ '==', 'id_index', 3 }} },
         op = 'select',
     },
-    replace = {
+    {
+        name = 'replace',
         func = 'crud.replace',
         args = {
             space_name,
             { 12, box.NULL, 'Ivan', 'Ivanov', 20, 'Moscow' },
```

Comment:
> if `replace()` is performed
> before `insert()`, an error will be received

Well, this logic is complicated and not easy to catch on. If one would change the name of a case, some tests may break, and it is not good.

I'm sorry that I have introduced this flaw in statistics test, but I think it's better to fix it by using different records (more specifically, records with different id) for inserts and replaces.

Reply:
Done.